superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/ag-grid/AGGridViz.tsx

```diff
@@ -448,6 +450,28 @@ export default function AGGridViz({
           }
           key="submit-file-to-assembly-line"
           data={selectedData.typeData.eml_path}
+          base_url={assemblyLineUrl}
+          disabled={selectedData.typeData.eml_path.length > SUBMISSION_LIMIT}
+          tooltip={
+            new Set(selectedData.typeData.eml_path).size > SUBMISSION_LIMIT
```

Comment:
Would it make sense to have a tooltip message even when the option is enabled? Or do we have a habit of only providing tooltips when something is disabled? 

If it makes sense to have a tooltip, I'd suggest something like "A new tab will open for each distinct EML path." (Just to warn users that they might be about to pop open 8 or 9 new tabs.

Reply:
The tooltip was suggested by Joel, and I believe it should only show up if the user selects more than 10 files to be submitted to AssemblyLine at the same time.

Reply:
Changed it such that the tooltip shows up both enabled or disabled, but with different messages.  

`tooltip={
            uniqueEmlPaths.size > SUBMISSION_LIMIT
              ? `You cannot submit more than ${SUBMISSION_LIMIT} eml files at a time.`
              : `A new tab will open for each distinct EML path submission.`
          }`